app/src/composables/use-asset-url.ts

```diff
@@ -0,0 +1,9 @@
+import { addTokenToURL } from '@/api';
+import { getRootPath } from '@/utils/get-root-path';
+import { computed } from 'vue';
+
+export function useAssetUrl(filename: string, isDownload?: boolean) {
+	return computed(() => {
+		return addTokenToURL(getRootPath() + `assets/${filename}` + (isDownload ? '?download' : ''));
+	}).value;
+}
```

Comment:
This feels rather odd as a composable, at least with the current way it's directly returning the value of a computed property 🤔 Not sure if this is idiomatic in terms of composables.

I think it should just be a util function, or alternatively as an exported function alongside `addTokenToURL` directly within the `@/api` file.

Reply:
Hey azrikahar, Thanks for your feedback! I had created it as Composables, since it was going to be a Computed. Utlitis would be here a normal JavaScript function without caching. Gladly I can change this of course!

Reply:
Makes sense! let's go with the utility route 👍